ppr-ui/src/resources/tableHeaders.ts

```diff
@@ -467,34 +467,13 @@ export const mhRegistrationTableHeaders: Array<BaseHeaderIF> = [
     value: 'statusType',
     display: true
   },
-  {
-    class: 'min-column-sm',
-    sortable: false,
-    text: 'Username',
-    value: 'registeringName',
-    display: true
-  },
   {
     class: 'min-column-mds',
     sortable: false,
     text: 'Submitting Party',
     value: 'registeringParty',
     display: true
   },
-  {
-    class: 'min-column-mds',
-    sortable: false,
-    text: 'Owners',
-    value: 'ownerNames',
-    display: true
-  },
-  {
-    class: 'min-column-mds',
-    sortable: false,
-    text: 'Folio/Reference Number',
-    value: 'clientReferenceId',
-    display: true
-  },
```

Comment:
Did Designs call for these columns to be removed?

Reply:
I followed the design in the ticket.. See https://projects.invisionapp.com/share/QW130SEIE7JB#/screens/469824642

Reply:
I believe the design gives the assumption that these columns are 'filtered', not removed entirely.
Please verify with Tracey.

Reply:
Confirmed with Tracey.  Fields were not put in design,  I have made the change... need to test when BCRGEG.. is working...

Reply:
@ChasBoucher Confirmed they are to remain and are filtered in the design? Or confirmed they are to be removed?